modules/@apostrophecms/schema/ui/apos/components/AposAttachment.vue

```diff
@@ -50,19 +56,78 @@ export default {
     },
     disabled () {
       return (this.limitReached || this.field.disabled);
+    },
+    moduleOptions() {
+      return window.apos.modules['@apostrophecms/file'];
     }
   },
   methods: {
     updated(items) {
       this.next = items;
     },
     validate(value) {
-      if (this.field.required && !value.data.length) {
+      if (this.field.required && !value) {
         return 'required';
       }
 
       return false;
-    }
+    },
+    async uploadMedia (event) {
+      this.$emit('upload-started');
+      const file = event.target.files[0];
+
+      const emptyDoc = await apos.http.post(this.moduleOptions.action, {
```

Comment:
This is an attachment editing field, for editing one attachment in the context of a schema. It never makes a new document, it is just a field. Why is this call being made?